db/migrate/20200420194305_set_defaults_caution_flags.rb

```diff
@@ -0,0 +1,11 @@
+class SetDefaultsCautionFlags < ActiveRecord::Migration[5.2]
+  def up
+    change_column_default :caution_flags,:title,'School engaged in misleading, deceptive, or erroneous practices'
+    change_column_default :caution_flags, :description,'VA has found that this school engaged in misleading, deceptive, or erroneous advertising, sales, or enrollment practices, and has taken action against it.'
+  end
+
+  def down
+    change_column_default :caution_flags,:title,nil
```

Comment:
nitpick, but there should be a space after each comma.  I'm surprised the linter doesn't say anything.